stats/sink.go

```diff
@@ -56,9 +56,11 @@ func (c *CounterSink) Add(s Sample) {
 func (c *CounterSink) Calc() {}
 
 func (c *CounterSink) Format(t time.Duration) map[string]float64 {
+	rps := c.Value / (float64(t) / float64(time.Second))
```

Comment:
I am wondering, how often is this calculated, what's the scale of `t`?

Reply:
Because a value of 1 divided by 0.5 seconds will produce `rps == 2` which is not optimal :sweat_smile:

Reply:
It is executed every 2 seconds currently: https://github.com/loadimpact/k6/blob/4f45b8fd009c846429e84804de1c019055c00a87/core/engine.go#L41
But yeah, that is something we have to fix in the new architecture as well...